web/static/js/phoenix_html.js

```diff
@@ -1,6 +1,13 @@
-function didHandleSubmitLinkClick(element){
+function isLinkToSubmitParent(element) {
+  const isLinkTag = element.tagName === 'A';
+        shouldSubmitParent = element.getAttribute('data-submit') === 'parent';
```

Comment:
Let's use `var` instead of `const` and we need to use it on both lines. :) Thank you!